README.md

```diff
@@ -1,30 +1,118 @@
+##### Initializing a Catalog
+You can initialize a DeltaCAT catalog from any new or existing Iceberg catalog. When using an existing catalog, you can
+immediately read and write any tables it contains. When creating a new catalog, you can immediately start adding and
+managing new tables.
+```python
+import deltacat as dc
+from deltacat import IcebergCatalog
+
 
+# Start by initializing DeltaCAT and registering all Catalogs you'd like to make available to your Ray application.
+# Ray will be initialized automatically via `ray.init()`.
+
+# Use top-level DeltaCAT APIs (e.g. `dc.create_table`) to manage tables in any registered catalog.
+# DeltaCAT APIs will act on a default catalog unless a different catalog name is explicitly specified.
+
+# Here, since only the `iceberg` data catalog is registered, it will become the default.
+# When initializing multiple catalogs, you can use the `default_catalog_name` param
+# to specify which catalog should be used by default.
+dc.init(
+    catalogs={
+        # the name of the DeltaCAT catalog is "iceberg"
+        "iceberg": dc.Catalog(
+            # Apache Iceberg implementation of deltacat.catalog.interface
+            impl=IcebergCatalog,
+            # kwargs for pyiceberg.catalog.load_catalog start here...
+            # the name of the Iceberg catalog is "example-iceberg-catalog"
+            name="example-iceberg-catalog",
+            # for additional properties see:
+            # https://py.iceberg.apache.org/configuration/
+            properties={"type": "glue"},
+        )
+    },
+)
 ```
-pip3 install virtualenv
-virtualenv test_env
-source test_env/bin/activate
-pip3 install -r requirements.txt
+The `iceberg` catalog is now registered as the default catalog for your Ray application and is ready to start receiving
+requests.
+
+##### Creating a New Table
+You can now create your first table in the `iceberg` catalog.
+
+```python
+import deltacat as dc
+
+from pyiceberg.schema import (
+    Schema,
+    NestedField,
+    DoubleType,
+    StringType,
+    TimestampType,
+    FloatType,
+)
+from pyiceberg.partitioning import PartitionSpec, PartitionField
+from pyiceberg.transforms import DayTransform, IdentityTransform
+from pyiceberg.table.sorting import SortField, SortOrder
+
+# define a simple native Iceberg Schema
+schema = Schema(
+    NestedField(
+        field_id=1, name="datetime", field_type=TimestampType(), required=True
+    ),
+    NestedField(field_id=2, name="symbol", field_type=StringType(), required=True),
+    NestedField(field_id=3, name="bid", field_type=FloatType(), required=False),
+    NestedField(field_id=4, name="ask", field_type=DoubleType(), required=False),
+)
+
+# define a native Iceberg partition spec
+partition_spec = PartitionSpec(
+    PartitionField(
+        source_id=1, field_id=1000, transform=DayTransform(), name="datetime_day"
+    )
+)
+
+# define a native Iceberg sort order
+sort_order = SortOrder(SortField(source_id=2, transform=IdentityTransform()))
```

Comment:
+1. I expect the overall format of these examples to change pretty soon too as we re-orient Iceberg as more of a synchronization target, rather than a direct DeltaCAT Catalog/Storage interface implementation like it is here.